components/HeroIcon/HeroIcon.styles.tsx

```diff
@@ -78,7 +78,7 @@ export const iconBaseStyle: IconBaseStyleType = {
   'triangle-up': 'w-09em scale-x-90 -rotate-90 mt-02em',
   download: 'w-09em',
   email: 'w-1em',
-  external: 'w-08em',
+  external: 'w-08em stroke-[2.5]',
```

Comment:
The diagonal arrow icon for external link is just the right arrow rotated, due to this it appears thinner than the other ones, so I'm increasing the stroke with a bit to make it match the apparent stroke with of the right arrows.